modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/helper/DLURLHelperImpl.java

```diff
@@ -441,6 +435,50 @@ private String _getDLFileVersionURLProviderURL(
 		return null;
 	}
 
+	private String _getFriendlyURL(
+		boolean appendVersion, FileEntry fileEntry, String previewURLPrefix,
+		String queryString) {
```

Comment:
The parameter sorting looks a bit weird (lexicographic order). Why not follow the same "logical" ordering as in `_getPreviewUuidURL`?